### DISCUSSION TITLE
Can mqtt virtual buttons be supported?

### DISCUSSION THREAD

**Discussion by wisonlau** [arendst/Tasmota#19566] [Support request]
For example, I have two mqtt topic:

1.button: /test/btn1
Responsible for controlling the button.
btn1 = 1,Status is on.
btn1 = 0,Status is off.

2.status: /test/btn1_status
Responsible for monitoring button status.

Can the button be displayed in the tasmota interface and display the status correctly?

**Comment by sfromis** [comment]
Short answer, Tasmota has no features for that.

Longer answer is that with Berry or scripting, you could likely do something to get such. Or you could config relays on unused pins, which would give you web UI buttons, and by default MQTT messages/commands when you turn them on/off. Tasmota additionally has a subscribe feature for MQTT payloads.

More generally, the deliberately rudimentary Tasmota web UI is not meant as a place to create a "device dashboard", and thus no "neat" features for such.
